index.html

```diff
@@ -0,0 +1,637 @@
+              </div>
+              <div class="d-flex flex-column more-main">
+                <img src="https://d.newsweek.com/en/full/1574186/house.jpg?w=95&h=95&f=05d0a25b64e3d92f1850f73140f17c0e" alt="">
+                <a href="#">40 Republicans Voted Against the 'Families First' Coronavirus Response Bill</a>
+                <p>"We all want to help Americans as we face the spread of coronavirus, but this complete rush job is not the way to do it," said Republican Representative Debbie Lesko.</p>
+
+              </div>
+
+            </div>
+
+          </div>
+          <div class="col-lg-3 col-md-12 opinion d-flex flex-column">
+            <h4 class="heading">OPINION</h4>
+
+            <img src="https://d.newsweek.com/en/full/83499/newt-gingrich.png?w=63&h=63&l=48&t=55&f=d8b4f8dfaad8a5f0f71bffcc6c0d6a77" alt="">
+            <p class="opinion-p">I Am in Italy Amid the Coronavirus Crisis. America Must Act Now—And Act Big</p>
+            <span>BY NEWT GINGRICH</span>
+            <hr>
+            <img src="https://d.newsweek.com/en/full/111050/craig-considine.png?w=63&h=63&f=86981e327051b13e3771dc08bb69f7f8" alt="">
+            <p class="opinion-p">By Parroting 'Chinese Coronavirus,' Republicans Are Promoting Racism</p>
+            <span>BY JUDY CHU</span>
+            <hr>
+            <img src="https://d.newsweek.com/en/full/1574439/dan-caldwell.png?w=63&h=63&f=9112a861ea096c0b55f6d98918261dad" alt="">
+            <p class="opinion-p">Coronavirus Is Exposing Trump's 'Strong Economy' for the Hoax It Is</p>
+            <span>BY ANGELA HANKS</span>
+            <hr>
+            <img src="https://d.newsweek.com/en/full/101658/jennifer-weiss-wolf.png?w=63&h=63&f=0b82c369922ff03b0e424910c2ec3167" alt="">
+            <p class="opinion-p">Don't Panic Over Coronavirus. That's Just What the President's Enemies Want</p>
+            <span>BY CHARLIE KIRK</span>
+            <hr>
+            <img src="https://d.newsweek.com/en/full/83499/newt-gingrich.png?w=63&h=63&l=48&t=55&f=d8b4f8dfaad8a5f0f71bffcc6c0d6a77" alt="">
+            <p class="opinion-p">Coronavirus and In-Person Voting Don't Mix. We Must Prepare for Digitized Elections</p>
+            <span>BY LEEZA GARBER</span>
+            <hr>
+            <img src="https://d.newsweek.com/en/full/83499/newt-gingrich.png?w=63&h=63&l=48&t=55&f=d8b4f8dfaad8a5f0f71bffcc6c0d6a77" alt="">
+            <p class="opinion-p">Trump Is Hurting the Economy by Linking Coronavirus to Immigration</p>
+            <span>BY IBRAHIM ALHUSSEINI</span>
+            <hr>
+
+            <h4>LATEST NEWS</h4>
+            <div class="congress">
+              <img src="https://d.newsweek.com/en/full/1574961/don-young.jpg?w=95&h=95&f=e61b82af5d45f0beeffdb3b7894d6fca" alt="">
+              <p>Congressman Attended NRA Fundraiser After Skipping COVID-19 Relief Vote</p>
+            </div>
```

Comment:
- [ ] The articles in this section should have a bold title and include a link to them. currently, you are using a `<p>` tag.

- [ ] You are missing some articles, this will affect your page in other screen sizes.